file_browser/src/file_browser/local/partials/1-elements/_buttons.scss

```diff
@@ -0,0 +1,59 @@
+[class*=regular-button] {
+    color: var(--white-color);
+    background-color: var(--color1-normal);
+    font-family: $font;
+    border: none;
+    border-radius: 5px;
+    flex: 1;
+    margin: 15px;
+    cursor: pointer;
+    height: 30px;
+    font-weight: bold;
+    &:hover {
+      background-color: var(--color1-dark);
+    }
+}
+
+.dialog-button {
+  color: var(--white-color);
+  background-color: var(--color1-normal);
+  font-family: $font;
+  border: none;
+  border-radius: 5px;
+  flex: 1;
+  margin: 15px;
+  cursor: pointer;
+  height: 30px;
+  font-weight: bold;
+  &:hover {
+    background-color: var(--color1-darker);
+  }
+}
```

Comment:
And after you update the selector with my comment above, you'll be able to remove all this content excelty the hover